src/app/core/feature-module/featured.service.ts

```diff
@@ -148,34 +145,6 @@ export class FeaturedObjectsService {
     }
   }
 
-  /**
-   * Fetches Array of Learning Objects that are not currently featured
-   *
-   * @returns {Promise<LearningObject[]>}
-   * @memberof LearningObjectService
-   */
-  getNotFeaturedLearningObjects(
-    query?: Query,
-  ): Promise<{ learningObjects: LearningObject[]; total: number }> {
-    let route = '';
-    if (query) {
-      const queryClone = Object.assign({}, query);
-      const queryString = querystring.stringify(queryClone);
-      route =
-        SEARCH_ROUTES.GET_PUBLIC_LEARNING_OBJECTS_WITH_FILTER(queryString);
-    } else {
-      route = SEARCH_ROUTES.GET_PUBLIC_LEARNING_OBJECTS;
-    }
-
-    return this.http
-      .get(route)
-      .pipe(catchError(this.handleError))
-      .toPromise()
-      .then((response: any) => {
-        return { learningObjects: response.objects, total: response.total };
-      });
-  }
-
```

Comment:
I believe this should still be located here or the admin dashboard will break.

Reply:
This is actually using a learning-objects search route. I would update the component that uses it and it should be okay. I just think the name of the function should be updated to match how we refer to it in the backend.